modules/Campaigns/tpls/WizardCampaignHeader.tpl

```diff
@@ -90,7 +90,7 @@
 					<tr{if $HIDE_CAMPAIGN_TYPE} style="display: none;"{/if}>
 						<!--
 					<td scope="col"><span sugar='slot5'>{$MOD.LBL_CAMPAIGN_START_DATE} </span sugar='slot'></td>
-					<td ><span sugar='slot5b'><input id='start_date' name='wiz_step1_start_date' title='{$MOD.LBL_CAMPAIGN_START_DATE}' onblur="parseDate(this, '{$CALENDAR_DATEFORMAT}');"  type="text" size='11' maxlength='10' value="{$CAMP_START_DATE}"> {sugar_getimage name="jscalendar" ext=".gif" alt=$APP.LBL_ENTER_DATE other_attributes='align="absmiddle" id="start_date_trigger" '} <span class="dateFormat">{$USER_DATEFORMAT}</span></span sugar='slot'></td>
+					<td ><span sugar='slot5b'><input id='start_date' name='wiz_step1_start_date' title='{$MOD.LBL_CAMPAIGN_START_DATE}' onblur="parseDate(this, '{$CALENDAR_DATEFORMAT}');"  type="text" size='11' maxlength='10' value="{$CAMP_START_DATE}"> <span class=suitepicon suitepicon-module-calendar" id="start_date_trigger"></span> <span class="dateFormat">{$USER_DATEFORMAT}</span></span sugar='slot'></td>
```

Comment:
please remove the dead codes (is it intend change or just a missed find+replace command?)

Reply:
I have removed the "dead" code from this file.